src/api.c

```diff
@@ -124,14 +128,17 @@ API const unsigned int seccomp_api_get(void)
 API int seccomp_api_set(unsigned int level)
 {
 	switch (level) {
-	case 1:
-		sys_set_seccomp_syscall(false);
-		sys_set_seccomp_flag(SECCOMP_FILTER_FLAG_TSYNC, false);
-		break;
+	case 3:
+		sys_set_seccomp_flag(SECCOMP_FILTER_FLAG_LOG, true);
+	/* fallthrough */
 	case 2:
 		sys_set_seccomp_syscall(true);
 		sys_set_seccomp_flag(SECCOMP_FILTER_FLAG_TSYNC, true);
```

Comment:
It seems as though we should force off the FLAG_LOG functionality in the case of API levels 1 and 2, yes?

Reply:
Hmm... you're right.

That means that the odd indentation of "fallthrough" doesn't matter because we can't fall through any longer.